scripts/deep_learning/test_classifier.py

```diff
@@ -0,0 +1,394 @@
+#         start = v[0][0]
+#         for i in range(1, len(v)):
+#             if v[i][0] - v[i-1][0] == 1:
+#                 curr.append(v[i][1])
+#             else:
+#                 max_occ = Counter(curr).most_common(1)[0][0]
+#                 new_v.append((start, max_occ))
+#                 curr = [v[i][1]]
+#                 start = v[i][0]
+#         max_occ = Counter(curr).most_common(1)[0][0]
+#         new_v.append((start, max_occ))
+#         picklist_map[k] = new_v
+#
+#     for k, v in true_map.items():
+#         new_v = []
+#         curr = [v[0][1]]
+#         start = v[0][0]
+#         for i in range(1, len(v)):
+#             if v[i][0] - v[i-1][0] == 1:
+#                 curr.append(v[i][1])
+#             else:
+#                 max_occ = Counter(curr).most_common(1)[0][0]
+#                 new_v.append((start, max_occ))
+#                 curr = [v[i][1]]
+#                 start = v[i][0]
+#         max_occ = Counter(curr).most_common(1)[0][0]
+#         new_v.append((start, max_occ))
+#         true_map[k] = new_v
+#
+#     final_pl_map = {}
+#     for k, v in picklist_map.items():
+#         for frame_no, pred in v:
+#             final_pl_map[(k, frame_no)] = pred
+#     final_true_map = {}
+#     for k, v in true_map.items():
+#         for frame_no, label in v:
+#             final_true_map[(k, frame_no)] = label
+#
+#     true = []
+#     preds = []
+#     for k in final_pl_map:
+#         preds.append(final_pl_map[k])
+#         true.append(final_true_map[k])
+#
+#     print(preds)
+#     print(true)
+#     total = 0
+#     correct = 0
+#     for p, t in zip(preds, true):
+#         correct += p==t
+#         total += 1
+#     # print(true_map)
+#     total_acc = correct / total
+#     print('###############################')
+#     print('Test Accuracy: {}'.format(
+#         total_acc
+#     )
+#     )
+#     print('###############################')
+#     # accs.append((epochs, total_acc))
+#     confusion_matrix = metrics.confusion_matrix(true, preds)
+#     conf_mat_norm = (confusion_matrix.astype('float') / confusion_matrix.sum(axis=1)[:, np.newaxis])
+#     conf_mat_norm = np.around(conf_mat_norm, decimals=2)
+#     # unique_names = unique_labels(actual_picklists_names, predicted_picklists_names)
+#     cm_display = metrics.ConfusionMatrixDisplay(confusion_matrix=conf_mat_norm,
+#                                                 display_labels=['red', 'green', 'blue', 'darkblue', \
+#                                                                 'darkgreen', 'orange', 'alligatorclip', 'yellow', \
+#                                                                 'clear', 'candle'])
+#
+#     cm_display.plot(cmap=plt.cm.Blues)
+#
+#     plt.xticks(rotation=90)
+#
+#     plt.tight_layout()
+#
+#     # plt.show()
+#
+#     plt.savefig(f'results_resnet9_test_classifier_grouped.png')
+#
+#     # print(accs)
+#
+# if __name__ == '__main__':
+#     # train()
+#     # test()
+#     test_per_pick()
```

Comment:
is there anything in this file?

Reply:
if it's just the main program part, then just uncomment that part i think, to make it clear